docs/userguides/transactions.md

```diff
@@ -0,0 +1,193 @@
+4. **Execution**: When enough signatures are collected, the transaction can be executed on-chain
+
+## Creating and Proposing Transactions
+
+You can create transactions very easily using your safe account like any other Ape account type:
+
+### Via Python API
+
+```python
+# Create a simple transfer
+# NOTE: This will fail to execute due to `submit=False`,
+#       but will successfully submit to the Safe Gateway
+safe.transfer("0xRecipientAddress", "1 ether", submit=False)
+```
+
+You can also do more complex contract interactions:
+
+```python
+# Load a contract
+token = Contract("0xTokenAddress")
+
+# Call a contract method
+# NOTE: This will fail to execute due to `submit=False`,
+#       but will successfully submit to the Safe Gateway
+token.transfer(
+    "0xRecipientAddress",
+    amount,
+    sender=safe,
+    submit=False,
+)
+```
+
+### Via CLI
+
+```bash
+# Propose a simple ETH transfer
+ape safe pending propose --to 0xRecipientAddress --value "1 ether"
+
+# Propose a contract interaction
+ape safe pending propose --to 0xTokenAddress --data 0x123abc...
+```
+
+## Listing Pending Transactions
+
+Once a transaction has been signed and proposed to the Safe Gateway, you can fetch them from the service
+
+### Via Python API
+
+```python
+# Get all pending transactions in the Safe Gateway's queue
+for tx, confirmations in safe.pending_transactions():
+    print(f"Nonce: {tx.nonce}, To: {tx.to}, Data: {tx.data}, Value: {tx.value}")
+    print(f"Confirmations: {len(confirmations)}/{safe.confirmations_required}")
+```
+
+### Via CLI
+
+```bash
+# List pending transactions
+ape safe pending list
+
+# Show more details
+ape safe pending list --verbosoe
+
+# Show confirmations for a transaction
+ape safe pending show-confs 0x123...abc
+```
+
+## Adding Confirmations to Transactions
+
+You can also add more confirmations (signatures) to proposed transactions in the Safe Gateway queue.
+
+### Via Python API
+
+```python
+# Find transaction by nonce
+for tx, _ in safe.pending_transactions():
+    if tx.nonce == 42:  # The nonce you're looking for
+        # Add signatures from local signers
+        signatures = safe.add_signatures(tx)
+
+        # Check if we have enough signatures to execute
+        if len(signatures) >= safe.confirmations_required:
+            print("Transaction ready to execute")
+```
+
+### Via CLI
+
+```bash
+# Approve by transaction hash
+ape safe pending approve 0x123abc...
+
+# Or approve by nonce (if only one transaction with that nonce)
+ape safe pending approve 42
+```
+
+## Executing Transactions
+
+When a transaction has enough signatures, it can then be executed on-chain.
+
+### Via Python API
+
+```python
+from ape import accounts
+
+# Get a regular account to submit the transaction
+submitter = accounts.load("my-account")
+
+# Find and execute a transaction that has enough signatures
+for tx, confirmations in safe.pending_transactions():
```

Comment:
is there a way to just get the tx by nonce directly?

Reply:
not pending ones, that's a good suggestion to add